tdrs-backend/tdpservice/data_files/util.py

```diff
@@ -52,11 +51,36 @@ def format_error_msg(x):
     ]
 
     # write beta banner
-    worksheet.write(row, col,
-                    "Error reporting in TDP is still in development." +
-                    "We'll be in touch when it's ready to use!" +
-                    "For now please refer to the reports you receive via email")
+    worksheet.write(
+        row, col,
+        "Please refer to the most recent versions of the coding " +
+        "instructions (linked below) when looking up items " +
+        "and allowable values during the data revision process"
+    )
+
+    row, col = 1, 0
+    worksheet.write_url(
+        row, col,
+        'https://www.acf.hhs.gov/ofa/policy-guidance/tribal-tanf-data-coding-instructions',
+        string='Tribal TANF Instructions'
+    )
+
     row, col = 2, 0
+    worksheet.write_url(
+        row, col,
+        'https://www.acf.hhs.gov/ofa/policy-guidance/acf-ofa-pi-23-04',
+        string='TANF / SSP-MOE (ACF-199 / ACF-209) Instructions'
+    )
+
+    row, col = 3, 0
+    worksheet.write_url(
+        row, col,
+        'https://tdp-project-updates.app.cloud.gov/knowledge-center/viewing-error-reports.html',
```

Comment:
this was the link provided in the ticket, but it currently `404`s

Reply:
This should start working as soon as we merge #2969